packages/teleport/src/Databases/Databases.tsx

```diff
@@ -51,18 +51,22 @@ export function Databases(props: State) {
     authType,
   } = props;
 
-  const isEmpty = attempt.status === 'success' && databases.length === 0;
+  const isEmpty =
+    (attempt.status === 'success' || attempt.status === 'processing') &&
+    databases.length === 0;
```

Comment:
i think this will show both the indicator and the empty state if it's processing?